pinot-core/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
```

Comment:
Does this need a licensing header?

Reply:
We ignore license check for `test/resources` but i will add the header for completeness.